### PR TITLE
Stars were created

### PR DESCRIPTION
**Pull request by KrisMur**
- [DEMO LINK](https://KrisMur.github.io/layout_stars/)
- [TEST REPORT LINK](https://KrisMur.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+* {
```
Comment (Olexandr6) [inline src/style.css:2]:
```suggestion
* {
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  height: 16px;
+  display: flex;
+}
```
Comment (Olexandr6) [inline src/style.css:28]: Delete this

Comment (Olexandr6) [reply src/style.css:28]: Find another, more straightforward way to implement this logic.

File: src/index.html [hunk 3]
```diff
@@ -8,6 +8,47 @@
...
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--5">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```
Comment (Olexandr6) [inline src/index.html:52]: [Checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md) 1st point

**Review by Olexandr6 — CHANGES_REQUESTED** [review]

**Comment by KrisMur** [comment]
made changes according to the comments

**Review by Olexandr6 — APPROVED** [review]
Good job👍

**Pull request closed without merging**